feeds.xsh

```diff
@@ -0,0 +1,24 @@
+import asyncio
+import threading
+import time
+from feedy import Feedy
+
+app = Feedy('feeds.shelve')
+
+@app.add('https://stackoverflow.com/feeds/tag?tagnames=xonsh&sort=newest')
+def stackoverflow_question(info, body):
```

Comment:
`feeds.shelve` keeps state. Feedy handles it.

It will display old ones on first start-up, though.